astropy/modeling/core.py

```diff
@@ -325,7 +325,7 @@ def __call__(self, **kwargs):
 
             if param.default is param.empty:
                 raise ModelDefinitionError(
-                    'The bounding_box method for {0} is not correctly '
```

Comment:
I'm a bit worried how painful these changes will be for https://github.com/astropy/astropy/pull/8769, maybe skip this file.

Reply:
Will do.